src/material/icon/icon-registry.ts

```diff
@@ -149,7 +149,7 @@ export class MatIconRegistry implements OnDestroy {
    * specified. The default 'material-icons' value assumes that the material icon font has been
    * loaded as described at http://google.github.io/material-design-icons/#icon-font-for-the-web
    */
-  private _defaultFontSetClass = ['material-icons'];
+  private _defaultFontSetClass = ['material-icons', 'mat-ligature-font'];
```

Comment:
I think that adding it to the defaults may break existing clients. We should document in `icon.md` how people can opt into it instead. A live example would be good as well.